File: backend/write_test_image.js
const fs = require('fs');
const b64 = 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR4nGNgYAAAAAMAASsJTYQAAAAASUVORK5CYII=';
fs.writeFileSync('backend/test_upload.png', Buffer.from(b64,'base64'));
console.log('wrote backend/test_upload.png');

File: backend/write_test_image.py
import base64
png_b64 = "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR4nGNgYAAAAAMAASsJTYQAAAAASUVORK5CYII="
with open('backend/test_upload.png','wb') as f:
    f.write(base64.b64decode(png_b64))
print('wrote backend/test_upload.png')
